mscore/exportmidi.cpp

```diff
@@ -79,6 +79,33 @@ void ExportMidi::writeHeader()
             }
 #endif
 
+      //--------------------------------------------
+      //    write track name
```

Comment:
I hate to be so picky, but the comment should really be “write track names”, not “write track name”.

Reply:
No problem. As you might guess I'm not native English and have no problems when people help to improve :-).

Reply:
Done. Thanks again.